sdk/core/Azure.Core/api/Azure.Core.netstandard2.0.cs

```diff
@@ -538,7 +538,7 @@ public sealed partial class ResourceIdentifier : System.IComparable<Azure.Core.R
         public ResourceIdentifier(string resourceId) { }
         public Azure.Core.AzureLocation? Location { get { throw null; } }
         public string Name { get { throw null; } }
-        public Azure.Core.ResourceIdentifier Parent { get { throw null; } }
+        public Azure.Core.ResourceIdentifier? Parent { get { throw null; } }
```

Comment:
Why does this need to be nullable? ResourceIdentifier is already a class and Parent can be null.

Reply:
move to this pr https://github.com/Azure/azure-sdk-for-net/pull/26237